cms-common/src/main/java/io/commercetools/sunrise/cms/CmsServiceException.java

```diff
@@ -0,0 +1,8 @@
+package io.commercetools.sunrise.cms;
+
+public class CmsServiceException extends RuntimeException {
+
+    public CmsServiceException(final String message, final Throwable cause) {
+        super(message, cause);
+    }
+}
```

Comment:
I would also add the constructor for just the throwable (without message). I find it convenient.